index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile Version Skeleton</title>
+</head>
+
+<body>
+    <section class="container">
+        <section class="pg-1">
+            <header>
+                <img class="img" src="img/time.png" alt="time">
+
+                <nav>
+                    <ul>
+                        <li><img src="img/cellular.png" alt="cellular"></li>
+                        <li><img src="img/wifi.png" alt="wifi"></li>
+                        <li><img src="img/capacity.png" alt="capacity"></li>
+                    </ul>
```

Comment:
Please remove the `time, wifi, cellular and capacity` they are not part of the design.